crates/bevy_time/src/time.rs

```diff
@@ -122,11 +126,26 @@ impl Time {
     }
 
     /// The time from startup to the last update in seconds
+    ///
+    /// If you need an f32 value, it is highly recommended to use [`Time::seconds_since_startup_f32_wrapped`]
+    /// instead of casting it. Casting to an f32 can cause floating point precision issues pretty fast.
```

Comment:
```suggestion
    /// If you need an `f32` value, it is highly recommended to use [`Time::seconds_since_startup_f32_wrapped`]
    /// instead of casting it. Casting to an `f32` can cause floating point precision issues pretty fast.
```